### ISSUE TITLE
Image variance collapses to zero

### ISSUE THREAD

**Issue by karlHolmquist** [NVlabs/DIODE#15]
Hi, thanks for the code, I just want to make sure whether what I'm currently encountering is due to parameter settings or the architecture I'm using.

I've made some modifications to the code so this will not be representative of the original training code but when I'm generating images using a pretrained resnest-101 the generated images quickly reaches a zero-variance (less than 0.01 after 100 iterations). Is this something that often happen or is this a problem that occur due to the differing architecture?

Some potential causes:
Different architecture (Resnest-101 + DeepLabv3)
All BN layers are used in the loss (I've tried to use the cosine annealing of the number of layers to include)
Loss balances (I've tried to completely remove the task loss and only use the task loss + include an image statistics loss matching the mean and variance to a N(0,1) distribution (after normalization))

Thanks beforehand for any insights

**Comment by karlHolmquist** [comment]
Figured it out, I had missed that I accidentally used weight decay which caused the variance to tend to zero.. Closing the issue
